### PR TITLE
Date format

### LINKED ISSUE #2303: [Bug] Date format
### Bug Description

image

Should there be a space betwee `,` and `2023`

### How to Reproduce

Add a Date column
Use this Date format: Month day, Year

### Expected Behavior

Month Day, Year

### Operating System

MacOS

### AppFlowy Version(s)

Main branch

### Screenshots

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by vuzzer**
### Feature Preview

This PR changes the previous date format from `Month day,Year` to `Month day, Year`. closes #2303

#### Screenshot
Capture d’écran 2023-05-01 à 23 19 28

#### PR Checklist

- [x] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/date_cell/date_editor.dart [hunk 1]
```diff
@@ -49,7 +49,7 @@ class _DateCellEditor extends State<DateCellEditor> {
       ),
       builder: (BuildContext context, snapshot) {
         if (snapshot.hasData) {
-          return _buildWidget(snapshot);
+          return  _buildWidget(snapshot) ;
```
Comment (a-wallen) [inline frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/date_cell/date_editor.dart:52]:
```suggestion
          return _buildWidget(snapshot);
```

No need for these whitespace changes.

File: frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/grid_page.dart [hunk 2]
```diff
@@ -83,7 +83,7 @@ class _GridPageState extends State<GridPage> {
             loading: (_) =>
                 const Center(child: CircularProgressIndicator.adaptive()),
             finish: (result) => result.successOrFail.fold(
-              (_) => const GridShortcuts(child: FlowyGrid()),
+              (_) => const  GridShortcuts(child: FlowyGrid() ),
```
Comment (a-wallen) [inline frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/grid_page.dart:86]:
```suggestion
              (_) => const GridShortcuts(child: FlowyGrid()),
```

remove whitespace changes please :)

File: frontend/appflowy_flutter/lib/plugins/database_view/application/field/type_option/type_option_context.dart [hunk 3]
```diff
@@ -143,7 +143,8 @@ class TypeOptionContext<T extends GeneratedMessage> {
   }
 
   set typeOption(T typeOption) {
-    _dataController.typeOptionData = typeOption.writeToBuffer();
+    _dataController.typeOptionData
+     = typeOption.writeToBuffer();
```
Comment (a-wallen) [inline frontend/appflowy_flutter/lib/plugins/database_view/application/field/type_option/type_option_context.dart:147]:
```suggestion
    _dataController.typeOptionData = typeOption.writeToBuffer();
```

remove whitespace changes

**Review by a-wallen — APPROVED** [review]

**Review by richardshiue — COMMENTED** [review]
The tests are failing, could you please update them at `frontend/rust-lib/flowy-database/src/services/field/type_options/text_type_option/text_tests.rs` and `frontend/rust-lib/flowy-database/src/services/field/type_options/date_type_option/date_tests.rs`? 

And also, if you have the time, could you also add the space in the other language files as well?

**Comment by vuzzer** [comment]
I deleted the assignment by mistake, @annieappflowy could you please reassign me?

**Pull request merged**